src/test/scala/tech/sourced/gemini/CassandraSparkSpec.scala

```diff
@@ -126,6 +127,9 @@ class CassandraSparkSpec extends FlatSpec
 
   object Cassandra extends Tag("Cassandra")
 
+  // such tests require bblfsh & feature extractors
+  object Integration extends Tag("Integration")
```

Comment:
👍, why not calling  it
```scala
object RequireBblfshAndFeatureExtractor
```
? 

That way reader of https://github.com/src-d/gemini/pull/105/files#diff-354f30a63fb0907d4ad57269548329e3R22 has additional benefit of not needing to think what excatly does "Integration" mean in this context.

Reply:
Integration it a common term which means tests require external dependencies.
I don't think `RequireBblfshAndFeatureExtractor` is a good name for it.
We may remove FeatureExtractor service (implementing it in scala) or add one more external dependency (community-detection-as-a-service for example). It shouldn't cause renaming of this tag.
Also it is `RequireCassandaAndBblfshAndFeatureExtractor` which doesn't look nice.
I really don't want it to become `RequireCassandaAndBblfshAndFeatureExtractorAndAnotherServiceAndOneMoreAndAnotherOne` later.

Reply:
> Integration it a common term which means tests require external dependencies.

True, but it's too broad an generic and depends on context a lot. I find readable variable names very useful, much more then comments near them and it's totally fine if sometimes things need to be renamed though refactoring.

From my side - a productive discussion here would include debating alternatives, rather then the approach though.